tools/icu/icu-generic.gyp

```diff
@@ -42,7 +42,6 @@
           }],
         ],
         'defines': [
-          'UCONFIG_NO_TRANSLITERATION=1',
```

Comment:
Is the v8 version visible as a parameter here? Perhaps we could parameterize this… same with the data.